core/templates/pages/topic-editor-page/editor-tab/topic-editor-tab.directive.ts

```diff
@@ -132,6 +133,59 @@ angular.module('oppia').directive('topicEditorTab', [
             $scope.topicNameExists = false;
             $scope.topicUrlFragmentExists = false;
             $scope.hostname = WindowRef.nativeWindow.location.hostname;
+            $scope.availableSkillsForDiagnosticTest = (
+              $scope.topic.getSkillSummariesThatAreNotInDiagnosticTest());
+            $scope.selectedSkillsForDiagnosticTest = (
+              $scope.topic.getDiagnosticTestSkillSummaries());
+            $scope.isDiagnosticTestSkillsDropdownPresent = false;
```

Comment:
Don't start variables with verbs (is/are). You can say diagnosticTestSkillsDropdownIsPresent (or, perhaps better, ...Shown), for example.

SImilarly below and elsewhere, please bear this in mind throughout.

Reply:
Updated